e2e/sendSheetFlow.spec.js

```diff
@@ -263,20 +250,20 @@ describe('Send Sheet Interaction Flow', () => {
     await Helpers.waitAndTap('send-asset-form-ETH-token');
   });
 
-  it('Should show Add Contact Screen after tapping Add Contact Button', async () => {
+  it.skip('Should show Add Contact Screen after tapping Add Contact Button', async () => {
     await Helpers.checkIfVisible('add-contact-button');
     await Helpers.waitAndTap('add-contact-button');
     await Helpers.checkIfVisible('wallet-info-input');
   });
 
-  it('Should do nothing on Add Contact cancel', async () => {
+  it.skip('Should do nothing on Add Contact cancel', async () => {
```

Comment:
are these broken on ios too? wondering the reason to `.skip`

Reply:
cc @tchayen

Reply:
Yea, it was Android-only case. I will restore them.